.github/workflows/integ-tests-with-security.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ windows-latest, macos-latest ]
```

Comment:
Why do we remove Mac?

Reply:
Mac was always failing anyway since it is not officially supported core artifact, so I decided to remove it since it is a useless check that is always failing